modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -694,9 +693,11 @@ else if (objectEntry.getStatus() != WorkflowConstants.STATUS_IN_TRASH) {
 				objectEntry.getObjectEntryId());
 		}
 
-		_deleteFileEntries(
-			Collections.emptyMap(), objectDefinition.getObjectDefinitionId(),
-			objectEntry::getValues);
+		_dlFileEntryLocalService.deleteFileEntries(
+			objectDefinition.getCompanyId(),
+			_classNameLocalService.getClassNameId(
+				objectDefinition.getClassName()),
+			objectEntry.getObjectEntryId());
```

Comment:
@pedro-oliveira446 apparently we do not delete files uploaded via computer but which are shown in documents and media (see [here](https://github.com/liferay-bpm/liferay-portal/blob/ee4e415a5ed610d712c3a83694b9c999905c26a4/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java#L2800-L2813)).

Did you check this case?

Reply:
Yes, in this case the DLFileEntry is saved without className and classPK, so it is not deleted.